Resources/config/services.xml

```diff
@@ -8,6 +8,8 @@
             <tag name="kernel.event_subscriber"/>
         </service>
 
+        <service id="httplug.collector.plugin_journal" class="Http\HttplugBundle\Collector\PluginJournal" public="false" />
```

Comment:
It is moved here because I did not want to send a list of Plugins to the ProfilerExtension. The overhead of having it here are minimum and I thought the code would be cleaner. 

It also explains https://github.com/php-http/HttplugBundle/pull/97#discussion_r71820163